wss-unified-agent.config

```diff
@@ -0,0 +1,17 @@
+excludes=**/its/sources/** **/kotlin-checks-test-sources/** **/*javadoc.jar
+fileSystemScan=False
+
+gradle.aggregateModules=True
+gradle.preferredEnvironment=wrapper
+gradle.resolveDependencies=True
+gradle.runPreStep=False
+
+maven.aggregateModules=False
+maven.downloadMissingDependencies=False
+maven.ignoredScopes=None
+maven.m2RepositoryPath=.m2/repository
+maven.resolveDependencies=False
+maven.runPreStep=False
+
+resolveAllDependencies=False
+wss.url=https://saas-eu.whitesourcesoftware.com/agent
```

Comment:
you are missing these here 
```

forceUpdate=true
checkPolicies=true
forceUpdate.failBuildOnPolicyViolation=true
```

Reply:
Thanks. The config file is now up to date.